### PR TITLE
Raids

### PR DESCRIPTION
**Pull request by jamierocks**
*The following are an exert from my mapping set, they are my own work*.

```
CL: asd net/minecraft/world/entity/projectile/WitherSkull
CL: asf net/minecraft/world/entity/raid/Raid
CL: asg net/minecraft/world/entity/raid/Raider
CL: ash net/minecraft/world/entity/raid/RaidManager
CL: asi net/minecraft/world/entity/vehicle/AbstractMinecart
```

(mapping order shows they are packaged together)

### CODE REVIEW

**Review by asiekierka — APPROVED** [review]

**Comment by jamierocks** [comment]
Nearly forgot - thoughts on `RaidState` vs `RaidManager`?

**Comment by asiekierka** [comment]
If it manages one raid, RaidState. If it manages all raids in a world, say, RaidManager. IMO

**Comment by jamierocks** [comment]
*Relevant discussion earlier*
```
21:40 <+Prospector> parent is PersistedState
21:42 <+Prospector> so it's for consistency
21:42 <+Prospector> but Manager may be a better name for that one too?
21:42 <+Prospector> the village one is called WorldVillageManager
```

**Comment by liach** [comment]
Ah jamie! Please don't kill my pr again!

**Comment by jamierocks** [comment]
@liach heh, sorry about that - feel a tad bad doing that twice :s

Having said that, I don't think it'll be too difficult to rebase yours for this (though I'd be happy for you just to make these changes in your pr - it's pretty much just the package different).

File: mappings/net/minecraft/entity/raid/RaiderEntity.mapping [hunk 1]
```diff
@@ -1,4 +1,4 @@
-CLASS asg net/minecraft/entity/mob/RaiderEntity
+CLASS asg net/minecraft/entity/raid/RaiderEntity
```
Comment (liach) [inline mappings/net/minecraft/entity/raid/RaiderEntity.mapping:1]: You should move other raiders as well.

**Review by ChloeDawn — APPROVED** [review]

**Review by shadowfacts — APPROVED** [review]

**Pull request merged**